code/modules/mob/living/say.dm

```diff
@@ -144,12 +144,25 @@ proc/get_radio_key_from_channel(var/channel)
 	else
 		speaking = get_default_language()
 
+	// Check if the speech requires dexterity, and if so, stop if can't use hands.
+	if(speaking && (speaking.flags & NEEDS_DEX))

```

Comment:
Figured a NEEDS_DEX flag would be nice for if we implemented sign language, as well, but okay.